app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
```

Comment:
@jcoyne Ok. The attr_reader can be removed. However, if I remove the `intitialize()`, the error above occurs.